### PR TITLE
Changed activity preset to new timeline layout

### PR DESCRIPTION
**Pull request by Nitwel**
**Dont merge this until the timeline interface also gets merged!**

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
Lets also add a migration and update the initial seeder to reflect this change.

**Comment by Nitwel** [comment]
Could you do this, I have 0 clue how this is done. 😄

**Comment by rijkvanzanten** [comment]
> `{{action}}d\`

This might be tricky i18n wise:

EN
`create` + `d` = `created` ✅
`upload` + `d` = `uploadd` ❌

**Comment by Nitwel** [comment]
Hmm, true. But I didnt knew any better implementation for now.
Maybe we could use the v-ext-display for this but then we would need to remove the icon or have multiple styles of the action display.

**Comment by rijkvanzanten** [comment]
> Maybe we could use the v-ext-display for this but then we would need to remove the icon or have multiple styles of the action display.

Then we'd still have the problem that we would have to translate these words in to all languages 🤔 

We could add `upload_past: 'Uploaded'` and `create_past: 'Created'` to the translation files, and then use `$t( action + '_past' )` to render the past value

**Comment by bjgajjar** [comment]
Hey @Nitwel 

There seem to be conflicts in the PR [(APP)](https://github.com/directus/app/pull/1589) you've created. Can you please check it and resolve?

File: migrations/db/seeds/CollectionPresetsSeeder.php [hunk 1]
```diff
@@ -18,21 +18,18 @@ public function run()
...
-                        'widths' => [
-                            'action' => 170,
-                            'action_by' => 170,
-                            'action_on' => 180,
-                            'collection' => 200,
-                            'item' => 200
-                        ]
+                    'timeline' => [
+                        'date' => 'action_on',
+                        'title' => '{{ action_by.first_name }} {{ action_by.last_name }} {{ action }}d',
```
Comment (rijkvanzanten) [inline migrations/db/seeds/CollectionPresetsSeeder.php]:
```suggestion
                        'title' => '{{ action_by.first_name }} {{ action_by.last_name }} ({{ action }})',
```

File: migrations/upgrades/schemas/20190520094300_use_timeline.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+              'view_type' => 'timeline',
+              'view_query' => json_encode([
+                'timeline' => [
+                    'sort' => '-action_on'
+                ]
+              ]),
+              'view_options' => json_encode([
+                'timeline' => [
+                    'date' => 'action_on',
+                    'title' => '{{ action_by.first_name }} {{ action_by.last_name }} {{ action }}d',
```
Comment (rijkvanzanten) [inline migrations/upgrades/schemas/20190520094300_use_timeline.php]:
```suggestion
                    'title' => '{{ action_by.first_name }} {{ action_by.last_name }} ({{ action }})',
```

File: src/schema.sql [hunk 3]
```diff
@@ -84,7 +84,7 @@ LOCK TABLES `directus_collection_presets` WRITE;
 
 INSERT INTO `directus_collection_presets` (`id`, `title`, `user`, `role`, `collection`, `search_query`, `filters`, `view_type`, `view_query`, `view_options`, `translation`)
 VALUES
-	(1,NULL,NULL,NULL,'directus_activity',NULL,NULL,'tabular','{\"tabular\":{\"sort\":\"-action_on\",\"fields\":\"action,action_by,action_on,collection,item\"}}','{\"tabular\":{\"widths\":{\"action\":170,\"action_by\":170,\"action_on\":180,\"collection\":200,\"item\":200}}}',NULL),
+	(1,NULL,NULL,NULL,'directus_activity',NULL,NULL,'timeline','{\"timeline\":{\"sort\":\"-action_on\"}}','{\"timeline\":{\"date\":\"action_on\",\"title\":\"{{ action_by.first_name }} {{ action_by.last_name }} {{action}}d\",\"content\":\"action_by\",\"color\":\"action\"}}',NULL),
```
Comment (rijkvanzanten) [inline src/schema.sql]:
```suggestion
	(1,NULL,NULL,NULL,'directus_activity',NULL,NULL,'timeline','{\"timeline\":{\"sort\":\"-action_on\"}}','{\"timeline\":{\"date\":\"action_on\",\"title\":\"{{ action_by.first_name }} {{ action_by.last_name }} ({{action}})\",\"content\":\"action_by\",\"color\":\"action\"}}',NULL),
```

**Comment by rijkvanzanten** [comment]
Lets fix the above strings. @bjgajjar the application side of this PR is done and merged

**Pull request merged**